release.nix

```diff
@@ -72,7 +72,8 @@ let
     unstable = pkgs.releaseTools.aggregate {
       name = "darwin-${pkgs.lib.nixpkgsVersion}";
       constituents =
-        [ jobs.stdenv.x86_64-darwin
+        [
+          jobs.stdenv.x86_64-darwin
```

Comment:
Ah, sorry, this is due to `nixpkgs-fmt`, I'll amend it to remove the unnecessary diff